src/registry.ts

```diff
@@ -113,6 +121,12 @@ export async function setEntry(
   entry: RegistryEntry,
   customOptions = {}
 ): Promise<void> {
+  // Assert the input is 64 bits.
+  const newint = BigInt.asUintN(64, entry.revision);
```

Comment:
I think this is fine but I would use the function as defined in the `BigInt.asUintN` documentation page literally: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt/asUintN 
That way I think it's more readable and we can cover it with a small unit test.

Side-note: I also kind of like the 'n' notion, like `10n`, as opposed to the explicit `BigInt(10)` type constructor

Reply:
Thanks, I'll make that change.

I like `10n` as well but it is only available in ECMA 2020